config_templates/gretel/transform/default.yml

```diff
@@ -16,18 +21,11 @@ models:
                   - person_name
                   - credit_card_number
                   - phone_number
-                  - us_social_security_number
```

Comment:
I think `us_social_security_number` should stay, right?

Reply:
yeah, i realized that after i deleted it. will add it back